plutus-pab/src/Plutus/PAB/Db/Memory/ContractStore.hs

```diff
@@ -67,7 +67,7 @@ handleContractStore = \case
         case result of
             Just s  -> pure s
             Nothing -> throwError (ContractInstanceNotFound instanceId)
-    Contract.GetActiveContracts -> do
+    Contract.GetContracts _ -> do
```

Comment:
You don't want to use the filter here?

Reply:
That's a good question. I hope @j-mueller will help to answer it.

The thing is that `Contract.PutStopInstance` handler is empty `pure ()` which means that we don't stop instances actually with this type of memory?

Reply:
Yes good point; you're right, it's fine then.

Reply:
Yes, we don't actually track this for the in-memory store (it's only supposed to be a debugging / demo tool)